man/dot-epi_spread.Rd

```diff
@@ -16,7 +21,7 @@ of total population in group $i$ (model will normalise if needed)}
 \item{p_susceptibility}{A matrix giving the probability that an individual
 in demography group $i$ is in risk (or susceptibility) group $j$.
 Each row represents the overall distribution of individuals in demographic
-group $i$ across risk groups, and each row *must sum to 1.0*.}
+group $i$ across risk groups, and each row \emph{must sum to 1.0}.}
```

Comment:
documentation does not render latex math mode. `\eqn{}` in `.solve_newton()` and `.solve_iterative()` are rendering correctly

Reply:
This should be fixed by `usethis::use_roxygen_md()`.

Reply:
Thanks, not sure why this is the case, as I have the markdown option enabled for Roxygen. Now using `\eqn` throughout.